miasm2/arch/aarch64/disasm.py

```diff
@@ -0,0 +1,27 @@
+from miasm2.core.asmbloc import asm_constraint, disasmEngine
+from miasm2.arch.aarch64.arch import mn_aarch64
+
+cb_aarch64_funcs = []
+
+
+def cb_aarch64_disasm(mn, attrib, pool_bin, cur_bloc, offsets_to_dis, symbol_pool):
+    for func in cb_aarch64_funcs:
+        func(mn, attrib, pool_bin, cur_bloc, offsets_to_dis, symbol_pool)
+
+
+class dis_aarch64b(disasmEngine):
+    attrib = 'b'
+
+    def __init__(self, bs=None, **kwargs):
+        super(dis_aarch64b, self).__init__(
+            mn_aarch64, self.attrib, bs, **kwargs)
+        self.dis_bloc_callback = cb_aarch64_disasm
+
+
+class dis_aarch64l(disasmEngine):
+    attrib = 'l'
+
+    def __init__(self, bs=None, **kwargs):
```

Comment:
This is exactly the same code that for `dis_aarch64b` and for some architecture.
May the code of `disasmEngine`, which seems to be an abstract class, should use `self.attrib` and `self.dis_bloc_callback` instead of taking it as an `__init__` argument.
Indeed, it is related to the class, not to the instance.

Then, this code can be rewritten:
```Python
class dis_aarch64l(disasmEngine):
    attrib = 'l'
    dis_bloc_callback = cb_aarch64_disasm

class dis_aarch64b(disasmEngine):
    attrib = 'b'
    dis_bloc_callback = cb_aarch64_disasm
```
It seems clearer for me, and, in addition, it avoids one to think that there is a special treatment in the `__init__` for *this* architecture.